mico-core/src/main/java/io/github/ust/mico/core/web/ServiceController.java

```diff
@@ -365,6 +378,16 @@ private MicoService getServiceFromDatabase(String shortName, String version) thr
         return serviceOpt.get();
     }
 
+    private List<MicoService> getServicesFromDatabase(String shortName) throws ResponseStatusException {
```

Comment:
Maybe change function name to `getVersionsOfServiceFromDatabase` or something similar?